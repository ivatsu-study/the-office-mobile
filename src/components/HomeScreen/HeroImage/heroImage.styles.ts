import { StyleSheet } from 'react-native'

const styles = StyleSheet.create({
  imageContainer: {
    width: '100%',
    height: 200,
    aspectRatio: 1 * 1.8,
  },
  image: {
    width: '100%',
    height: '100%',
    resizeMode: 'cover',
  },
})

export default styles
