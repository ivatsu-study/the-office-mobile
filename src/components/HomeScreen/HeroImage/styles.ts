import { StyleSheet } from 'react-native'

const styles = StyleSheet.create({
  imageContainer: {
    width: 200,
    height: 200,
    aspectRatio: 1 * 1.8
  },
  image: {
    width: '100%',
    height: '100%',
    resizeMode: 'cover'
  }
})

export default styles
