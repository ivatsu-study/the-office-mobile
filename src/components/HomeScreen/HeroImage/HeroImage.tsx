import { Image, View } from 'react-native'

import OfficeMainImage from '../../../../assets/office.png'
import styles from './heroImage.styles'

const HeroImage: React.FunctionComponent = () => (
  <View style={styles.imageContainer}>
    <Image style={styles.image} source={OfficeMainImage} />
  </View>
)

export default HeroImage
