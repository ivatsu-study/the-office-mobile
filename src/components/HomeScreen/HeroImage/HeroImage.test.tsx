import React from 'react'
import renderer from 'react-test-renderer'
import HeroImage from './HeroImage'

describe('<HeroImage />', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<HeroImage />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
