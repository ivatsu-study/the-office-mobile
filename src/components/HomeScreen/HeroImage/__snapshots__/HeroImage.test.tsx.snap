// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeroImage /> renders correctly 1`] = `
<View
  style={
    Object {
      "aspectRatio": 1.8,
      "height": 200,
      "width": 200,
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "height": "100%",
        "resizeMode": "cover",
        "width": "100%",
      }
    }
  />
</View>
`;
